code/modules/core_implant/cruciform/modules.dm

```diff
@@ -69,6 +70,7 @@
 	if(ishuman(implant.wearer))
 		var/mob/living/carbon/human/H = implant.wearer
 		dna = H.dna
+		stats = H.stats
```

Comment:
That's possible what smth will (or already) affect dead body stats.
It's better to make copy of it (values only, not mods).